controllers/reconcilers/kserve_networkpolicy_reconciler.go

```diff
@@ -19,6 +19,7 @@ import (
 	"context"
 	"github.com/go-logr/logr"
 	kservev1beta1 "github.com/kserve/kserve/pkg/apis/serving/v1beta1"
+	"github.com/kserve/kserve/pkg/constants"
```

Comment:
I would recommend to separate imports for better readability as a standard best practice.
Also sorting affects package initialization, first third party then local ones etc.

Reply:
@skonto Yeah I was having the same thoughts. In other projects I use `gci`, so it autoformats things:

```
  gci:
    sections:
      - standard
      - default
      - blank
      - prefix(github.com/opendatahub-io/opendatahub-operator)
      - blank
      - dot
    skip-generated: false
    custom-order: true
```

though that would be too much for the scope of this PR. @Jooho if you folks agree I can donate stuff like that in another PR.

> sorting affects package initialization

TIL :)

Reply:
Yes it is more like a comment for a separate PR.

Reply:
Yes, a separate PR would be good for build/chores